stream_alert/alert_processor/outputs.py

```diff
@@ -188,27 +187,26 @@ def dispatch(self, **kwargs):
             'severity': 'critical',
             'custom_details': details
         }
-        values_json = json.dumps({
+        data = {
             'routing_key': creds['routing_key'],
             'payload': payload,
             'event_action': 'trigger',
             'client': 'StreamAlert'
-        })
+        }
+        headers = None
+        verify = True
 
-        resp = self._request_helper(creds['url'], values_json)
+        resp = self._post_request(creds['url'], data, headers, verify)
         success = self._check_http_response(resp)
 
-        if not success:
-            response_value = json.load(resp)
-            error_message = response_value['error']['message']
-            detailed_errors = response_value['error']['errors']
+        if not success and resp:
```

Comment:
Can we have the `_check_http_response` method handle this error logging instead of doing it in a bunch of disparate places? You should be able to do something like this from the base class so the message is still context specific:

```
LOGGER.error('Encountered an error while sending to %s: %s\n%s',
    self.__service__,
    response['message'],
    response['errors'])
```